src/components/sections/Features/Features.tsx

```
@@ -1,45 +1,97 @@
+    <section className={styles.features} id="features">
+      <div className={styles.container}>
+        <div className={styles.grid}>
+          <div className={styles.feature}>
+            <div className={styles.iconWrapper}>
+              <svg
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)